src/main/java/com/providencegeeks/service/EventServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.providencegeeks.service;
+
+import com.providencegeeks.model.Event;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by mdiwale on 10/26/17.
+ */
+@Service
+public class EventServiceImpl implements EventService {
+
+  private EventDao eventDao;
+
+  public EventServiceImpl(EventDao eventDao) {
+    this.eventDao = eventDao;
+  }
+
+  @Override
+  public List<Event> getSortedEvents() {
```

Comment:
I think we should be more specific in this method.  Something like `getEventsSortedByTime`?

I'm not sure if this will be a long term viable strategy, but as we don't know the future and what the requirements of this service will be yet, we should probably avoid any premature refactoring.